scripts/ci/ci_before_install.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+set -xeuo pipefail
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export KUBERNETES_VERSION=${KUBERNETES_VERSION:=}
+# Required for K8s v1.10.x. See
+# https://github.com/kubernetes/kubernetes/issues/61058#issuecomment-372764783
+if [[ ! -z "${KUBERNETES_VERSION}" ]]; then
+    sudo mount --make-shared /
```

Comment:
Would this work for local development?

Reply:
I am yet looking into the Kubernetes tests. I think Kubernetes tests are fairly difficult at this stage to reproduce locally until we make a docker-compose based solution. This is a known problem that we decided for now to defer to next step "make the kubernetes CI tests better". I am happy to discuss and implement it in a better way :)

Reply:
BTW, I have a PR almost ready for running the k8s tests inside docker-compose. I need to fix a couple of things and then I'll publish it.

Reply:
Whoa! @gerardo  - that would be fantastic if we could have it and merge it together!

Reply:
Sweet

Reply:
@gerardo - I think I have the PR in almost mergable state. I have all the tests green except the Kubernetes ones. Any chance to take a look at your PR so that I can incorporate it ? (I can also fix the current way of running the kubernetes tests if the docker-compose kubernetes needs still some time :) ).

Reply:
@dimberman -> This is the WIP branch from @gerardo https://github.com/gerardo/airflow/tree/kind-ci. I think it will have to be rebased / updated quite a lot after we merge the docker image change but I am also happy to review it once this is done.

Reply:
@dimberman 
The kubernetes part is left out for now - it still works in Travis and we will be able to address it later on but for now it should be fine to leave it as is.